CodeEditModules/Modules/ShellClient/src/Live.swift

```diff
@@ -46,16 +46,16 @@ public extension ShellClient {
                             subject.send(completion: .finished)
                             return
                         }
-                        if let line = String(data: data, encoding: .utf8)?
```

Comment:
could you revert the the format as it was before? It's a good practice to go in a new line in this case